app/src/main/res/layout/app_bar.xml

```diff
@@ -62,15 +62,15 @@
         <View
             android:layout_width="1dp"
             android:layout_height="match_parent"
-            android:background="?attr/colorPrimary" />
+            android:background="?attr/white" />
```

Comment:
Same issue here, the parameter value can not be found during the build and will fail the build.  Just leave the original value unchanged